server/package.json

```diff
@@ -12,32 +13,31 @@
   "author": "",
   "license": "ISC",
   "devDependencies": {
-    "@types/express": "^4.17.2",
-    "@types/express-http-proxy": "^1.5.12",
-    "@types/express-session": "^1.15.16",
-    "@types/helmet": "0.0.45",
-    "@types/jest": "^26.0.14",
-    "@types/node": "^13.7.4",
-    "@types/passport": "^1.0.2",
-    "@types/redis": "^2.8.16",
+    "@types/express": "^4.17.11",
+    "@types/express-http-proxy": "^1.6.1",
+    "@types/express-session": "^1.17.3",
+    "@types/jest": "^26.0.20",
+    "@types/node": "^14.14.25",
+    "@types/passport": "^1.0.6",
+    "@types/redis": "^2.8.28",
     "@types/tunnel": "0.0.1",
-    "jest": "^25.1.0",
-    "nodemon": "^2.0.2",
-    "ts-jest": "^26.4.1",
-    "typescript": "^3.8.2"
+    "jest": "^26.6.3",
+    "nodemon": "^2.0.7",
+    "ts-jest": "^26.5.1",
+    "typescript": "^4.1.4"
   },
   "dependencies": {
-    "connect-redis": "^4.0.4",
+    "connect-redis": "^5.1.0",
     "dotenv": "^8.2.0",
     "express": "^4.17.1",
-    "express-http-proxy": "^1.6.0",
-    "express-session": "^1.17.0",
-    "fp-ts": "^2.5.3",
+    "express-http-proxy": "^1.6.2",
+    "express-session": "^1.17.1",
+    "fp-ts": "^2.9.5",
     "fs": "0.0.1-security",
-    "helmet": "^3.21.2",
-    "io-ts": "^2.2.0",
+    "helmet": "^4.4.1",
+    "io-ts": "^2.2.14",
     "io-ts-promise": "^2.0.2",
-    "openid-client": "^3.13.0",
+    "openid-client": "^3.15.10",
```

Comment:
Det finnes en versjon 4 av det her, men det var helt umulig å få til med en tunneling proxy vi har for å komme ut av poden for å nå microsoft. så oppgraderte til siste versjon 3